test/integrations/destinations/criteo_audience/dataDelivery/business.ts

```diff
@@ -0,0 +1,352 @@
+                  attemptNum: 1,
+                  destinationId: 'dummyDestinationId',
+                  dontBatch: false,
+                  jobId: 2,
+                  secret: {},
+                  sourceId: 'dummySourceId',
+                  userId: 'dummyUserId',
+                  workspaceId: 'dummyWorkspaceId',
+                },
+                statusCode: 200,
+              },
+            ],
+          },
+        },
+      },
+    },
+  },
+  {
+    id: 'criteo_audience_business_2',
+    name: 'criteo_audience',
+    description: '[Business]:: Test for mobile type audience to remove users with success response',
+    feature: 'dataDelivery',
+    module: 'destination',
+    version: 'v1',
+    input: {
+      request: {
+        body: generateProxyV1Payload(
+          {
+            JSON: {
+              data: {
+                type: 'ContactlistAmendment',
+                attributes: {
+                  operation: 'remove',
+                  identifierType: 'madid',
+                  internalIdentifiers: false,
+                  identifiers: [
+                    'sample_madid',
+                    'sample_madid_1',
+                    'sample_madid_2',
+                    'sample_madid_10',
+                    'sample_madid_13',
+                    'sample_madid_11',
+                    'sample_madid_12',
+                  ],
+                },
+              },
+            },
+            params,
+            headers,
+            method,
+            endpoint: 'https://api.criteo.com/2022-10/audiences/34893/contactlist',
+          },
+          [
+            {
+              jobId: 3,
+              attemptNum: 1,
+              userId: 'dummyUserId',
+              sourceId: 'dummySourceId',
+              destinationId: 'dummyDestinationId',
+              workspaceId: 'dummyWorkspaceId',
+              secret: {},
+              dontBatch: false,
+            },
+          ],
+        ),
+      },
+    },
+    output: {
+      response: {
+        status: 200,
+        body: {
+          output: {
+            status: 200,
+            message: 'Request Processed Successfully',
+            response: [
+              {
+                error: '""',
+                metadata: {
+                  attemptNum: 1,
+                  destinationId: 'dummyDestinationId',
+                  dontBatch: false,
+                  jobId: 3,
+                  secret: {},
+                  sourceId: 'dummySourceId',
+                  userId: 'dummyUserId',
+                  workspaceId: 'dummyWorkspaceId',
+                },
+                statusCode: 200,
+              },
+            ],
+          },
+        },
+      },
+    },
+  },
+  {
+    id: 'criteo_audience_business_3',
+    name: 'criteo_audience',
+    description: '[Business]:: Test for mobile type audience where audienceId is invalid',
+    successCriteria: 'Should return a 400 status code with an error message',
```

Comment:
What error message can be expected please add more details using description and successCriteria so that people can understand integration better from analysing test cases